--- 0037_auto_20180814_1433.py	2024-09-06 11:01:41.709068696 +0800
+++ 0037_auto_20180814_1433_after.py	2024-09-25 12:58:52.971064069 +0800
@@ -10,7 +10,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        # migrations.swappable_dependency(settings.AUTH_USER_MODEL),
         ('mooring', '0036_auto_20180808_1528'),
     ]
 
