--- 0044_auto_20180823_0951.py	2024-09-06 11:01:41.713068655 +0800
+++ 0044_auto_20180823_0951_after.py	2024-09-25 13:01:26.213795154 +0800
@@ -10,7 +10,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        # migrations.swappable_dependency(settings.AUTH_USER_MODEL),
         ('mooring', '0043_admissionsbooking_noofconcessions'),
     ]
 
