--- 0124_auto_20190402_1119.py	2024-09-06 11:01:41.713068655 +0800
+++ 0124_auto_20190402_1119_after.py	2024-09-25 13:12:34.132901333 +0800
@@ -10,7 +10,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        # migrations.swappable_dependency(settings.AUTH_USER_MODEL),
         ('mooring', '0123_admissionsbooking_created_by'),
     ]
 
